### PR TITLE
video: adding face detection and person detection samples for beta

### PR DESCRIPTION
**Pull request by czahedi**

### CODE REVIEW

File: video/cloud-client/analyze/video_detect_person_beta.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            # Each segment includes timestamped objects that include
+            # characteristic - -e.g.clothes, posture of the person detected.
+            # Grab the first timestamped object
+            timestamped_object = track.timestamped_objects[0]
+            box = timestamped_object.normalized_bounding_box
+            print("\tBounding box:")
+            print("\t\tleft  : {}".format(box.left))
+            print("\t\ttop   : {}".format(box.top))
+            print("\t\tright : {}".format(box.right))
+            print("\t\tbottom: {}".format(box.bottom))
```
Comment (nnegrey) [inline video/cloud-client/analyze/video_detect_person_beta.py]: Does Bounding box need to be tabbed and the below double tabbed?

Comment (czahedi) [reply video/cloud-client/analyze/video_detect_person_beta.py]: Happy to do that if you think it makes it more readable

File: video/cloud-client/analyze/video_detect_faces_beta.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            print(
+                "Segment: {}s to {}s".format(
+                    track.segment.start_time_offset.seconds
+                    + track.segment.start_time_offset.nanos / 1e9,
+                    track.segment.end_time_offset.seconds
+                    + track.segment.end_time_offset.nanos / 1e9,
+                )
+            )
+
+            # Each segment includes timestamped objects that include
```
Comment (dizcology) [inline video/cloud-client/analyze/video_detect_faces_beta.py]: objects --> faces

File: video/cloud-client/analyze/video_detect_faces_beta.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            # characteristics of the face detected.
+            # Grab the first timestamped object
```
Comment (dizcology) [inline video/cloud-client/analyze/video_detect_faces_beta.py]: object --> face

File: video/cloud-client/analyze/video_detect_faces_beta.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            timestamped_object = track.timestamped_objects[0]
+            box = timestamped_object.normalized_bounding_box
```
Comment (dizcology) [inline video/cloud-client/analyze/video_detect_faces_beta.py]: for face detection are there two versions of the bounding boxes?  one typically smaller than the other?  (vision API's face detection has that)

File: video/cloud-client/analyze/video_detect_faces_beta.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START video_detect_faces_beta]
+import io
+from google.cloud import videointelligence_v1p3beta1 as videointelligence
+
+
+def detect_face(local_file_path="path/to/your/video-file.mp4"):
```
Comment (dizcology) [inline video/cloud-client/analyze/video_detect_faces_beta.py]: maybe plural `detect_faces`?

File: video/cloud-client/analyze/video_detect_person_beta.py [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+            # poses, or hair color.
+            print("\tAttributes:")
+            for attribute in timestamped_object.attributes:
+                print(
+                    "\t\t{}:{} {}".format(
+                        attribute.name, attribute.value, attribute.confidence
+                    )
+                )
+
+            # Landmarks in person detection include body parts
```
Comment (dizcology) [inline video/cloud-client/analyze/video_detect_person_beta.py]: maybe list a few landmarks in the comment, like left_shoulder, right_knee etc.

File: video/cloud-client/analyze/video_detect_person_beta.py [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+            print("\tLandmarks:")
+            for landmark in timestamped_object.landmarks:
+                print(
+                    "\t\t{}: {} (x={}, y={})".format(
+                        landmark.name,
+                        landmark.confidence,
+                        landmark.point.x,
```
Comment (dizcology) [inline video/cloud-client/analyze/video_detect_person_beta.py]: are these normalized coordinates too, and (x, y)=(0, 0) at the upper left?  there are different conventions out there so perhaps it is beneficial to include that in the comment here.

File: video/cloud-client/analyze/video_detect_faces_gcs_beta_test.py [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+import os
+
+import video_detect_faces_gcs_beta
+
+RESOURCES = os.path.join(os.path.dirname(__file__), "resources")
+
+
+def test_detect_faces(capsys):
+    input_uri = "gs://cloud-samples-data/video/googlework_short.mp4"
```
Comment (dizcology) [inline video/cloud-client/analyze/video_detect_faces_gcs_beta_test.py]: googlework_tiny is used in the tests of person_detection.  perhaps use it here too?

Comment (czahedi) [reply video/cloud-client/analyze/video_detect_faces_gcs_beta_test.py]: The googlework_tiny video doesn't register any faces so unfortunately the test fails. I had to use this one. I could instead use this one in person_detection?

File: video/cloud-client/analyze/video_detect_person_beta.py (hunk 1 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_person_beta.py]: Did it in the most recent commit

File: video/cloud-client/analyze/video_detect_faces_beta.py (hunk 2 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_faces_beta.py]: Fixed in most recent commit

File: video/cloud-client/analyze/video_detect_faces_beta.py (hunk 3 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_faces_beta.py]: Fixed in most recent commit

File: video/cloud-client/analyze/video_detect_faces_beta.py (hunk 5 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_faces_beta.py]: Fixed in most recent commit

File: video/cloud-client/analyze/video_detect_person_beta.py (hunk 6 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_person_beta.py]: Fixed in most recent commit

File: video/cloud-client/analyze/video_detect_faces_beta.py (hunk 4 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_faces_beta.py]: I don't believe so. The Node canonicals didn't show this functionality.

File: video/cloud-client/analyze/video_detect_person_beta.py (hunk 7 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_person_beta.py]: This I'm not sure of and I'll look into it

File: video/cloud-client/analyze/video_detect_person_beta.py (hunk 1 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_person_beta.py]: Welp the linter got upset so I'm going to revert this

File: video/cloud-client/analyze/video_detect_faces_beta.py (hunk 4 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_faces_beta.py]: Looks like timestsmped_object is a repeated array so technically there could be more boxes but there's no other proto field with a bounding box for face: https://github.com/googleapis/java-video-intelligence/blob/master/proto-google-cloud-video-intelligence-v1p3beta1/src/main/proto/google/cloud/videointelligence/v1p3beta1/video_intelligence.proto

File: video/cloud-client/analyze/video_detect_person_beta.py (hunk 7 above)
Comment (czahedi) [reply video/cloud-client/analyze/video_detect_person_beta.py]: Yes, going to add this comment to both person samples: https://github.com/googleapis/java-video-intelligence/blob/master/proto-google-cloud-video-intelligence-v1p3beta1/src/main/proto/google/cloud/videointelligence/v1p3beta1/video_intelligence.proto#L445

**Pull request merged**